### DISCUSSION TITLE
How can I upgrade to 0.20.0 without losing existing playlist?

### DISCUSSION THREAD

**Discussion by Lady-Fitzgerald** [FreeTubeApp/FreeTube#4938] [Q&A]
I'm using Linux Mint 19.3 and 21.1 Cinnamon. I tried updating to version 0.20.0 and lost all the videos in my playlist. I had to do a Timeshift restore back to 0.19.2 to get my playlist back. I'm also getting error messages galore and I can't download comments. How the heck can I update without losing my existing videos in my playlist and start getting comments back?

Jeannie

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @Lady-Fitzgerald, we're sorry to hear that you lost your playlists data when upgrading to v0.20.0. This isnt done by FT because we dont touch these files during an upgrade. 

Did you maybe switch from flatpak to something else? Or for something else to flatpak. That would explain why it suddenly vanish but the data is defintly there, see link below.

What you can do is backup your playlist data and import it if its gone again after the upgrade. 

See link for data location: https://docs.freetubeapp.io/usage/data-location/

Comments are broken on previous versions of FT because YT messed around with it. 

> [!NOTE]
> Comments replies partially work for the Local API on v0.20.0 see https://github.com/FreeTubeApp/FreeTube/issues/4928 but a fix is on its way!
>
> Comments are broken on the Invidious side and requires a fix on their side.
